app/controllers/solidus_subscriptions/api/v1/subscriptions_controller.rb

```diff
@@ -6,6 +6,20 @@ module V1
       class SubscriptionsController < BaseController
         protect_from_forgery unless: -> { request.format.json? }
 
+        include SolidusSubscriptions::SubscriptionLineItemBuilder
```

Comment:
I didn't understand why this module is included. 🤔

Reply:
Good catch 👍 
I forgot to remove it.

Initially, I was using `create_subscription_line_item`, but then I removed it to stay with the `update` behavior.